rest_framework-stubs/schemas/openapi.pyi

```diff
@@ -17,11 +17,19 @@ class DRFOpenAPIInfo(TypedDict, total=False):
     version: str
     description: str
 
+class externalDocumentationObject(TypedDict):
```

Comment:
`class ExternalDocumentationObject`

Reply:
False. Fixed!

Reply:
`description` can be missing: 
![image](https://user-images.githubusercontent.com/2741256/126776077-698e6ee3-6dad-4bb3-a171-fcbda093680b.png)